Packs/Whois/Integrations/Whois/Whois_test.py

```diff
@@ -64,3 +66,35 @@ def cleanup():
     assert_results_ok()
     tmp.seek(0)
     assert 'connected to' in tmp.read()  # make sure we went through microsocks
+
+
+TEST_QUERY_RESULT_INPUT = [
+    (
+        {'contacts': {'admin': None, 'billing': None, 'registrant': None, 'tech': None},
+         'raw': ['NOT FOUND\n>>> Last update of WHOIS database: 2020-05-07T13:55:34Z <<<']},
+        'rsqupuo.info',
+        False
+    ),
+    (
+        {'status': ['clientUpdateProhibited (https://www.icann.org/epp#clientUpdateProhibited)'],
+         'updated_date': [datetime.datetime(2019, 9, 9, 8, 39, 4)],
+         'contacts': {'admin': {'country': 'US', 'state': 'CA', 'name': 'Google LLC'},
+                      'tech': {'organization': 'Google LLC', 'state': 'CA', 'country': 'US'},
+                      'registrant': {'organization': 'Google LLC', 'state': 'CA', 'country': 'US'}, 'billing': None},
+         'nameservers': ['ns1.google.com', 'ns4.google.com', 'ns3.google.com', 'ns2.google.com'],
+         'expiration_date': [datetime.datetime(2028, 9, 13, 0, 0), datetime.datetime(2028, 9, 13, 0, 0)],
+         'emails': ['abusecomplaints@markmonitor.com', 'whoisrequest@markmonitor.com'], 'raw': [
+            'Domain Name: google.com\nRegistry Domain ID: 2138514_DOMAIN_COM-VRSN'],
+         'creation_date': [datetime.datetime(1997, 9, 15, 0, 0)], 'id': ['2138514_DOMAIN_COM-VRSN']},
+        'google.com',
+        True
+    )
+
+]
+
+
+@pytest.mark.parametrize('whois_result, domain, expected', TEST_QUERY_RESULT_INPUT)
+def test_query_result(whois_result, domain, expected):
```

Comment:
lets add the following test cases: 
 - `Found`
 - `raw` is empty
 - `raw` is not a list, but a dict or str